pywren/storage/storage.py

```diff
@@ -25,6 +25,19 @@ def __init__(self, config):
         else:
             raise NotImplementedError(("Using {} as storage backend is" +
                                        "not supported yet").format(config['storage_backend']))
+    def head_object(self, key):
+        """
+        Retrieves metadata for given key.
+        The metadata dict must have Contentlength and Etag as keys
```

Comment:
Is it `Contentlength` or `ContentLength` ? (or does it not matter)

Reply:
I think it's `ContentLength`